### ISSUE TITLE
please help to resolve the problem: 'tuple' object has no attribute 'layer' in line : x = Conv2D(16, (3, 3), activation='relu', padding='same')(input_img)

### ISSUE THREAD

**Issue by usthbstar** [keras-team/keras#13943]
# -*- coding: utf-8 -*-
import numpy as np
#import keras
import tensorflow.keras
#from keras.datasets import mnis
from sklearn.model_selection import train_test_split
from tensorflow.keras.layers import Dense, Dropout, Flatten
from tensorflow.keras.layers import Conv2D, MaxPooling2D, Conv2DTranspose, Reshape , UpSampling2D
from keras.models import Model, Input
from keras.layers import  Concatenate, concatenate
from keras.backend.tensorflow_backend import set_session
from keras.backend.tensorflow_backend import clear_session
from keras.backend.tensorflow_backend import get_session
import tensorflow
from PIL import Image
from array import array
import gc
import os
from keras.utils import np_utils

from keras.preprocessing.image import load_img
from keras.preprocessing.image import img_to_array
from keras.preprocessing.image import array_to_img

img_rows, img_cols = 28, 28
img = load_img('mdct3.jpeg')
img_array = img_to_array(img)
X_Train=img_array
X_Train = np.array(X_Train).astype('float32')
X_Train /=255 
# The encoding process
input_img = Input(shape=(28,28,1))
x = Conv2D(16, (3, 3), activation='relu', padding='same')(input_img)
x = MaxPooling2D(pool_size = (2, 2), padding='same')(x)
x = Conv2D( 8, kernel_size = (3, 3), activation='relu', padding='same')(x)
x = MaxPooling2D(pool_size = (2, 2), padding='same')(x)
x = Conv2D( 8, (3, 3), activation='relu', padding='same')(x)
encoded = MaxPooling2D(pool_size = (2, 2), padding='same')(x)

**Comment by tilakrayal** [comment]
@usthbstar ,

You are trying to imported layers from tensorflow.keras while other functions you have imported from keras. Can you please either import your layers from keras or try importing other functions from tensorflow.keras which might work.Thanks!
